tests/test_transaction.py

```diff
@@ -42,6 +42,12 @@ def test_transaction_execute_raw(db, col, docs):
     assert err.value.error_code == 10
 
 
+def test_transaction_list(db, col, docs):
+    transactions = db.list_transactions()
+    assert isinstance(transactions, list)
+    # TODO: Add proper transaction test here
```

Comment:
todo

Reply:
added in [3d6c241](https://github.com/ArangoDB-Community/python-arango/pull/283/commits/3d6c24117f51bf59c68b5c7aeea6f32ad018365a)